.drone.star

```diff
@@ -712,7 +711,7 @@ def dockerRelease(ctx, arch):
       build() + [
       {
         'name': 'dryrun',
-        'image': 'plugins/docker:18.09',
+        'image': 'plugins/docker:latest',
```

Comment:
Let's hope this doesn't break anything randomly. :b

Reply:
We are using that almost everywhere because releases do not happen on a regular basis an this is the only way you get some fixes :-(